frontend/src/models/behaviors.js

```diff
@@ -26,3 +27,111 @@ export class RandomCardBehavior {
     return null;
   }
 }
+
+export class BasicRuleBasedBehaviour {
+  announcementToMake(possibleAnnouncements) {
+    // ToDo announce by "goodies"
+    /**
+     * for instance
+     * only one fehlsuit
+     * high number of trumps
+     * high trumps, such as 2x ten of hearts
+     * blank ace and starting
+     */
+    return null;
+  }
+
+  cardToPlay(hand, trick, memory) {
+    let baseCard = trick.baseCard();
+    if (!baseCard) {
+      /** It's our turn. Decide how to deal with cards */
+      return this.startingRule(hand, memory);
+    }
+    if (!baseCard.isTrump()) {
+      /** We need to serve a fehl card */
+      return this.fehlRule(hand, trick, memory);
+    }
+    // ToDo how to play if not starting or mustn't serve fehl
+    // i'm thinking of something working with expectation value
+    return sample(playableCards(hand.cards, baseCard));
+  }
+
+  startingRule(hand, memory) {
+    for (const suit of [suits.clubs, suits.spades, suits.hearts]) {
+      if (hand.hasBlankAce(suit) && !memory.fehlSuitPlayedBefore(suit)) {
+        return hand.fehlCardsBySuit(suit)[0];
+      }
+    }
+    // ToDo check if we know with whom we play and if we want to play a strategy
+    return sample(playableCards(hand.cards, undefined));
+  }
+
+  fehlRule(hand, trick, memory) {
+    let baseCard = trick.baseCard();
+    if (hand.mustServeSuit(baseCard.suit)) {
+      let fehlCards = hand.fehlCardsBySuit(baseCard.suit);
+      let highest = fehlCards[0];
+      let lowest = fehlCards.reverse()[0];
+
+      if (memory.fehlSuitPlayedBefore(baseCard.suit)) {
+        return lowest;
+      } else {
+        if (
+          trick.highestCard().card.value < highest.value &&
+          highest.value === 11 &&
+          fehlCards.length < 4
```

Comment:
After all changes, this remains the only thing I don't fully understand. Can you explain the logic in full terms to me just so I understand?

To me this reads like this:

* another player opened a trick with a non-trump
* we have to serve non-trump
* we've got a non-trump card that's got a higher value than the highest card in the current trick
* that specific card we have is an ace, by the way
* and we've got less than 4 trumps on our hands
* as a result, we play the highest trump card we can serve - which is always gonna be an ace

This sounds like a lot of very specific conditions and I'm not sure where we're heading with that.

Reply:
Okay, so this seems big but is quite easy to explain.

**You can only win the trick if you're having the highest card.**

As soon as an Ace is played, there no way to win the trick -> play lowest card.
If you don't have an ace to win the trick -> lowest card.
Since we're playing a sharp doko here, we have at most 6 cards of a suit (obviously it's different for hearts, but I don't want to cover everthing in rules).
If you have 4 cards of a suit, there is definately a player that doesn't have the suit (5 cards you know, two players left), so it's not possible to win the trick -> lowest card.
If it's three, there is a chance that the remaining two cards are split into the remaining players hands and you have a chance of winning the trick.

Obviously we don't count teams into this right now, which is something I want to add later, when affinities are implemented 😁

Reply:
I added new tests to explain this better: 8efde77207564c13b0fdf2d4df52a7b4edcef6ff